bitbots_body_behavior/src/bitbots_body_behavior/main.dsd

```diff
@@ -90,18 +93,18 @@ $LocalizationAvailable
             YES --> @EndTimer + name:reorientation, @StartTimer + name:reorientation_pause + duration:%behavior/body/reorientation_pause_duration
             NO --> $BallSeen
                 YES --> $BallInDefensiveArea
-                    YES --> #GoAndKickBallMapGoal
-                    NO -->  $BallClose
+                    YES --> $ClosestToBallNoGoalie
```

Comment:
The goalie only goes to the ball if it is in a certain area. If the ball is 3m away from the goalie it will not go there, and therefore none will go to the ball. The offense player would then be supporter.

Reply:
But in my understanding the goalie and e.g. the defense player both go to the ball if it's in the area near the goal in the current implementation.

Reply:
This should then maybe take the action of the goalie into account.

Reply:
Or we simply check if the ball is in the penalty area and leave it for the goalie if it's in there.

Reply:
I added a decision to see if the goalie is handling the ball